docs/DataWorkflows/DataCycle.md

```diff
@@ -0,0 +1,671 @@
+
+```
+# this command will return a URL you can open in any browser and a token you can use to sign in
+globusconnect -start &
+```
+
+3. Using a new or existing conda environment (see [here](https://pennlinc.github.io/docs/pmacs#logging-in-to-pmacs-lpc) for how to activate conda on PMACs), install the [CLI](https://docs.globus.org/cli/) using `pip` and login with `globus login`.
+
+4. Visit [https://docs.globus.org/how-to/get-started/](https://docs.globus.org/how-to/get-started/) to access the File Manager, as in the Local Disk instructions, to start transferring data.
+
+
+# Curating BIDS Datasets
+
+
+BIDS curation can be a frustrating process. This guide describes best practices for
+the curation process using a local filesystem. We divide this process into multiple
+stages.
+
+
+## Preparing your environment
+
+If you are curating data on a Penn cluster, you will need to set up a conda
+[environment](https://pennlinc.github.io/docs/cubic#installing-miniconda-in-your-project-the-hard-way) for your project. You will need to install [datalad](Datalad.md) and CuBIDS
+for the rest of the steps in this workflow. To do so, create a conda
+environment.
+
+If you've already got `conda` setup for environment management, please skip this
+step and instead directly download `CuBIDS` [here](https://bids-bond.readthedocs.io/en/latest/). After that, you can head over to _Stage 0_.
+
+Otherwise, the easiest way to get your CUBIC project user started is to download
+the project initialization script.  Immediately upon getting access to
+your project, log in as the project user and run:
+
+```shell
+$ wget https://raw.githubusercontent.com/PennLINC/TheWay/main/scripts/cubic/cubic-setup-project-user.sh
+$ bash cubic-setup-project-user.sh ${HOME}
+```
+
+[Download and install CuBIDS](https://bids-bond.readthedocs.io/en/latest/index.html). Note that this environment must be activated for the rest of the steps.
+
+You'll also need to install BIDS Validator:
+
+```shell
+$ conda install nodejs
+$ npm install -g bids-validator
+$ which bids-validator #to make sure it installed
+```
+
+## Stage 0: Organization
+
+The process begins when data arrives on one of our servers. While it may not
+be possible to document everything that has happened to the data prior to its
+arrival to our server, the goal of the organization step is to document as
+much as we can about the process of receiving the data, and any initial
+changes we might make to it. The person in charge of organizing a data set
+should document any changes they make in a *Data Narrative* along with any
+relevant scripts used to a git-tracked repository.
+
+Create a project root directory. Initially we suggest keeping one copy of your
+initial data that will remain untouched as a backup during early stages of
+the workflow. This is not necessary if your data is archived elsewhere online.
+In the project root create a directory structure similar to this:
+
+```
```

Comment:
I think it would be good to link out to [`paper-template`](https://github.com/PennLINC/paper-template) for this. If that template doesn't have a good proposed organization, we can change it.